R/snapshot.R

```diff
@@ -976,21 +975,31 @@ renv_snapshot_filter_report_missing <- function(missing, type) {
 
   postamble <- c(
     "Packages must first be installed before renv can snapshot them.",
-    "Consider installing these packages using `renv::install()`.",
     if (type %in% "explicit")
       "If these packages are no longer required, consider removing them from your DESCRIPTION file."
     else
       "Use `renv::dependencies()` to see where this package is used in your project."
   )
   renv_pretty_print(
-    values = csort(unique(missing)),
+    values = unique(missing),
```

Comment:
Do we still want to sort here (not csort)?

Reply:
Ooops yes. And since this is a common need (and it'd be nice to keep all uses of `sort()` in pretty printing), I've made a note to add a sort argument to `renv_pretty_print()`.